EvotoApi/Areas/Management/ManagementApiAreaRegistration.cs

```diff
@@ -19,6 +19,12 @@ public override void RegisterArea(AreaRegistrationContext context)
                 "mana/{controller}/{action}/{id}",
                 new { action = "Index", id = UrlParameter.Optional }
             );
+
+            context.MapRoute(
```

Comment:
The route wouldn't work without this, and I have minimal knowledge of how all the routing is handled on this project - there seems to be multiple different approaches.